index.html

```diff
@@ -1,58 +1,147 @@
+			background-image: url('./images/galaxy.jpg');
+			background-repeat: no-repeat;
+			background-size: cover;
+		}
+		
+		.infoparts {
+			width: 1200px;
+			display: flex;
+			justify-content: center;
+			flex-wrap: wrap;
+		}
+		
+		.textbox {
+			width: 310px;
+			height: 120px;
+			color: white;
+			font-weight: lighter;
+			line-height: 1.5em;
+			background-color: rgba(0, 0, 0, 0.6);
+			margin: 0px 10px;
+			padding: 10px;
+			border-radius: 4px;
+		}
+		
+		.moreinfo {
+			width: 1010px;
+			height: 70px;
+			color: white;
+			font-weight: lighter;
+			line-height: 1.5em;
+			background-color: rgba(0, 0, 0, 0.6);
+			margin-top: 10px;
+			padding: 10px;
+			border-radius: 4px;
+		}
+		
+		.moreinfo a {
+			color: white;
+		}
+		
+		#container {
+			display: flex;
+			flex-flow: wrap;
+			flex-wrap: wrap;
+			align-content: center;
+			justify-content: center;
+			flex-direction: row;
+			margin: 0 !important;
+			flex-basis: auto;
+		}
+		
+		#question {
+			width: 100px;
+			height: 30px;
+			color: white;
+			font-weight: lighter;
+			margin-bottom: 10px;
+			background-color: rgba(0, 0, 0, 0.6);
+			border-radius: 3px;
+			display: flex;
+			align-content: center;
+			justify-content: center;
+			align-items: center;
+		}
+		
+		#question:hover {
+			cursor: pointer;
+		}
+		
+		#infobox {
+			display: none;
+		}
+		
+		img {
+			width: 100%;
+			margin: 20px;
+		}
+		
+		h1 {
+			font-size: 3em;
+			color: white;
+			padding-top: 80px;
+			/*background-color: rgba(183, 58, 98, 0.89);*/
+		}
 	</style>
 </head>
 
 <body>
 	<!--<img style ="width: 80px; height: 80px; margin-top:10px" src ="./images/logo.png">-->
-    <h1 >The Coding Space Projects</h1>
-   <div id="container">
-	<div>
-		<a href="./scratch"><img style ="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
-	</div>
-	<div>
-		<a href="http://woofjs.com/create"><img  style ="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'"/></a>
+	<h1>The Coding Space Projects</h1>
+	<div id="container">
+		<div>
+			<a href="./scratch"><img style="width:300px; height: 300px" id="banner1" src="./images/scratch2.png" onmouseover="this.src='./images/scratchhover.png'" onmouseout="this.src='./images/scratch2.png'" /></a>
+		</div>
+		<div>
+			<a href="http://woofjs.com/create"><img style="width:300px; height: 300px" id="banner3" src="./images/woof2.png" onmouseover="this.src='./images/woofhover.png'" onmouseout="this.src='./images/woof2.png'" /></a>
+		</div>
+		<div>
+			<a href="./web"><img style="width:300px; height: 300px" id="banner2" src="./images/web2.png" onmouseover="this.src='./images/webhover.png'" onmouseout="this.src='./images/web2.png'" /></a>
+		</div>
+		<div class="infoparts">
+			<div id="question">Learn More</div>
```

Comment:
This button is a little small and possibly hard to notice. Maybe increasing the size could help?

These explanations is particularly important for people to notice, especially when they're new to The Coding Space so we might want to make this information more front-and-center rather then only show up when you click a small button. 

However, we don't want to overwhelm new users on their first visit to our site or program. What do you think? Is there a way we can accomplish both?